bokehjs/src/lib/core/util/templating.ts

```diff
@@ -113,11 +113,11 @@ export function get_value(raw_name: string, data_source: ColumnarDataSource, i:
   }
 }
 
-export function replace_placeholders(str: string, data_source: ColumnarDataSource, i: Index, formatters?: Formatters, special_vars: Vars = {}): string {
-
+export function replace_placeholders(str: string, data_source: ColumnarDataSource, i: Index, formatters?: Formatters, special_vars: Vars = {}): string | Node[]  {
   // this handles the special case @$name, replacing it with an @var corresponding to special_vars.name
   str = str.replace(/@\$name/g, (_match) => `@{${special_vars.name}}`)
 
+  let unsafe = false
```

Comment:
For a user, "safe" as a format means "I am *stipulating* this content is safe (to me) and therefore does not require escaping". Using the name "unsafe" here to also mean "no escaping" seems potentially confusing. How about 
```
let unescaped = false  // or escaped = true
```

 instead?

Reply:
I changed it to `has_html`, as it is what this variable really conveys.